pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -687,12 +687,6 @@
     )
     private String resourceUsageTransportClassName = "";
 
-    @FieldContext(
```

Comment:
What was the rationale for making this not configurable?

Reply:
@jerrypeng while reviewing the previous PR(https://github.com/apache/pulsar/pull/10008), @codelipenghui and @315157973 gave feedback that the topic name is internal implementation detail that the user doesn't need to be exposed to. I did not have a strong reason to go the other way, so made this change. We can always make it configurable in the future, if we find that there is a use-case for it.